go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/decred/go-socks
+
+go 1.12
```

Comment:
I don't see any reason to require go 1.12 as I don't see any code that actually requires it.  Thus, I believe this should be `go 1.11` .